lib/nmatrix/math.rb

```diff
@@ -364,6 +364,37 @@ def cov(opts={})
     deviation_scores.transpose.dot(deviation_scores) / denominator
   end
 
+  # Raise a square matrix to a power. Be careful of buffer overflows!
```

Comment:
I mean a case where you assign 300 to a `char` in C and that doesnt do what you want it to.

That's numeric overflow?